### PR TITLE
Create account creation and network files for Performance Hub.

### LINKED ISSUE #623: Performance hub networking config
## User Story

As a mp engineer
I need to create the network configuration for the performance hub
So that the application can communicate

This will create a pre-production and production environment for performance hub 

## Definition of done

- [ ] networking json created
- [ ] networking json created and workflow run
- [ ] readme has been updated
- [ ] user docs have been updated
- [ ] another team member has reviewed
- [ ] tests are green

## Reference

[How to write good user stories](https://www.gov.uk/service-manual/agile-delivery/writing-user-stories)

### PR DESCRIPTION
**Pull request by zuriguardiola**
Amend cidr-allocation file to allocate CIDRs required for a new Performance Hub account. Add new account and networking json files to create account, set up security baselines, and networking once PR is merged.

Resolves #623

### CODE REVIEW

**Review by donmasters — APPROVED** [review]
LGTM

File: environments-networks/hmpps-preproduction.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "cidr": {
+    "transit_gateway": "10.233.0.128/26",
```
Comment (donmasters) [inline environments-networks/hmpps-preproduction.json:3]: Is this number correct ?

**Review by donmasters — CHANGES_REQUESTED** [review]

File: environments-networks/hmpps-preproduction.json (hunk 1 above)
Comment (zuriguardiola) [reply environments-networks/hmpps-preproduction.json]: Just changed it to the correct CIDR

**Review by donmasters — DISMISSED** [review]
LGTM

**Review by donmasters — APPROVED** [review]
LGTM

**Review by philhorrocks — APPROVED** [review]

**Pull request merged**